views/about.hbs

```diff
@@ -13,14 +13,19 @@
   </div>
 </div>
 <div class="container-fluid text-center">
-  <div class="team-row text-center">
-    <div class="col-sm-10">
+  <div class="team-row">
+    <div class="col-sm-10" id="team">
       {{#each team}}
-      <div class="team-box">
+      <div class="team-box" data-toggle="collapse" data-target="#{{id}}" onclick="(e) => {console.log(e);}">
         <img src={{img}} alt={{name}} class="img-fluid rounded-circle" style="height: 160px; width: 160px;">
         <p class="lead">{{name}}</p>
       </div>
+      <div id="{{id}}" class="collapse" data-parent="#team">
+        <div class="card-body team-description">
+          {{description}}
+        </div>
+      </div>
```

Comment:
This seems like a good candidate for a partial and a snapshot test

Reply:
@pheeria I will do this in a new branch once this branch is merged. I already tagged myself in the trello card and moved it to 'active'.